crates/circuit/src/dag_circuit.rs

```diff
@@ -4787,6 +4765,12 @@ impl DAGCircuit {
         &self.qargs_interner
     }
 
+    /// Returns an mutable view of the Interner used for Qargs
+    #[inline(always)]
+    pub fn qargs_interner_mut(&mut self) -> &mut Interner<[Qubit]> {
```

Comment:
I think this is something we were trying not to expose since it'd make it possible to bring `DAGCircuit` into an inconsistent state (e.g. someone swaps this `Interner` with an empty one).

Reply:
This was a mistake to include, I was doing it in `disjoint_utils` (mostly to be lazy) and it wasn't meant to be included in this PR. I'll remove it.

Reply:
Removed in: https://github.com/Qiskit/qiskit/pull/14313/commits/961e479b1070b6f4442d3257e03d2cb5a6a6c2cc